chainerrl/experiments/train_agent.py

```diff
@@ -88,6 +88,9 @@ def train_agent(agent, env, steps, outdir, max_episode_len=None,
 
     # Save the final model
     save_agent(agent, t, outdir, logger, suffix='_finish')
+    if evaluator:
+        if evaluator.save_best_so_far_agent and not evaluator.saved:
+            save_agent(agent, "best", outdir, logger, suffix='')
```

Comment:
Saving the agent as "best" when no evaluation is performed is confusing. I think it is better not to save at all.